Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/StateSynchronization/StateSynchronizationObserver.cs

```diff
@@ -11,37 +11,35 @@ namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
     /// <summary>
     /// This class observes changes and updates content on a spectator device.
     /// </summary>
-    public class StateSynchronizationObserver : Singleton<StateSynchronizationObserver>
+    public class StateSynchronizationObserver : LocatableDeviceNetworkManager<StateSynchronizationObserver>
```

Comment:
>LocatableDeviceNetworkManager [](start = 48, length = 29)

This adds a concept of spatial alignment to these state synchronization classes. It may make more sense for this localization logic to be handled through its own command registry compared to existing by default.

Reply:
I created a new base type NetworkManager that StateSynchronizationObserver, StateSynchronizatoinBroadcaster, and HolographicCameraNetworkManager can all inherit from. LocatableDevice is now a separate class that just deals with that aspect.